Enterprise-CMCS/eAPD
# 4331 [Bug] Other State Expense validation triggering on Cancel

Resolves #4331

### Description
The Bug: When opening an other state expense subform, the category field is automatically focused. This results in having the validation trigger when trying to click Cancel. Meaning to actually close the subform (right after creating it) you have to click the Cancel button twice.

Solution: Removed autofocus.

### Chromatic Link
https://www.chromatic.com/build?appId=61d5b948cf6f17003a12bf77&number=1286

### Automated test cases written

| Given | When | Then | Type (jest, tap, cypress) |
| ----- | ---- | ---- | ------------------------- |
| Non Personnel State Expense Form | Open | Can cancel form with one click | cypress |

### Steps to manually verify this change

1. Navigate to State staff and Expenses
2. Create a new other state expense subform
3. Try to immediately close the form by clicking Cancel
4. Verify that the form closes

### This pull request is ready to code review when

      tests, refactor
      screenreader, text scaling) OR an exemption is documented

### This pull request is ready to test when


### This pull request is ready to review when QA has


### This pull request can be merged when
